src/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect } from "react";
+import { action } from "@storybook/addon-actions";
+import { Story } from "@storybook/react/types-6-0";
+
+import MultiSelect, { MultiSelectProps } from "./MultiSelect";
+import useMultiSelect from "../../hooks/useMultiSelect";
+
+const items = [
+  {
+    id: "art",
+    name: "Art",
+  },
+  {
+    id: "architecture",
+    name: "Architecture",
+  },
+  {
+    id: "art_history",
+    name: "Art History",
+  },
+  {
+    id: "design",
+    name: "Design",
+    // Children array will only be renderd in a "dialog" variant
+    children: [
+      {
+        id: "fashion",
+        name: "Fashion",
+      },
+      {
+        id: "ux",
+        name: "User Experience",
+      },
+    ],
+  },
+  {
+    id: "business",
+    name: "Business",
+  },
+  {
+    id: "advertising",
+    name: "Advertising",
+  },
+  {
+    id: "government_and_law",
+    name: "Government and Law",
+  },
+  {
+    id: "civil_rights",
+    name: "Civil Rights",
+  },
+  {
+    id: "history",
+    name: "History",
+  },
+  {
+    id: "atlases",
+    name: "Atlases",
+  },
+  {
+    id: "cartography",
+    name: "Cartography",
+  },
+  {
+    id: "geology",
+    name: "Geology",
+  },
+];
+
+export const MultiSelectListboxStory: Story<MultiSelectProps> = (args) => {
+  // Example with custom hook useMultiSelect.
+  const { onChange, onClear, selectedItems } = useMultiSelect({
+    multiSelectId: args.id,
+    items,
+  });
+
+  // Hack to get storybook's action tab to log state change when selectedItems state changes.
+  useEffect(() => {
+    if (Object.keys(selectedItems).length !== 0) {
+      action("onChange")(selectedItems);
+    }
+  }, [selectedItems]);
+  //passing {..args} break component because it will pass onMixecStateChange and onApply
```

Comment:
You don't need this comment. It's understandable if args isn't used. If you keep it, add a space between "//" and "passing", capitalize "passing, and fixing typo on `onMixecStateChange`.

Reply:
removed the comment